openshift/e2e-tests-openshift.sh

```diff
@@ -47,173 +54,118 @@ function install_strimzi(){
-  oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-${MAISTRA_VERSION}/deploy/maistra-operator.yaml
+  # Install the ServiceMesh Operator
+  oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/release-${SERVING_VERSION}/openshift/servicemesh/operator-install.yaml
 
-  # Wait until the Operator pod is up and running
-  wait_until_pods_running istio-operator || return 1
+  # Wait for the istio-operator pod to appear
+  timeout 900 '[[ $(oc get pods -n openshift-operators | grep -c istio-operator) -eq 0 ]]' || return 1
 
-  # Workaround for MAISTRA-670
-  oc delete validatingwebhookconfiguration istio-operator.servicemesh-resources.maistra.io
+  # Wait until the Operator pod is up and running
+  wait_until_pods_running openshift-operators || return 1
 
-  # Deploy Istio
+  # Deploy ServiceMesh
+  oc new-project $SERVICEMESH_NAMESPACE
+  oc apply -n $SERVICEMESH_NAMESPACE -f https://raw.githubusercontent.com/openshift/knative-serving/release-${SERVING_VERSION}/openshift/servicemesh/controlplane-install.yaml
   cat <<EOF | oc apply -f -
 apiVersion: maistra.io/v1
-kind: ServiceMeshControlPlane
-metadata:
-  name: minimal-multitenant-cni-install
-  namespace: istio-system
-spec:
-  istio:
-    global:
-      multitenant: true
-      proxy:
-        autoInject: disabled
-      omitSidecarInjectorConfigMap: true
-      disablePolicyChecks: false
-    istio_cni:
-      enabled: true
-    gateways:
-      istio-ingressgateway:
-        autoscaleEnabled: false
-        type: LoadBalancer
-      istio-egressgateway:
-        enabled: false
-      cluster-local-gateway:
-        autoscaleEnabled: false
-        enabled: true
-        labels:
-          app: cluster-local-gateway
-          istio: cluster-local-gateway
-        ports:
-          - name: status-port
-            port: 15020
-          - name: http2
-            port: 80
-            targetPort: 80
-          - name: https
-            port: 443
-    mixer:
-      enabled: false
-      policy:
-        enabled: false
-      telemetry:
-        enabled: false
-    pilot:
-      # disable autoscaling for use in smaller environments
-      autoscaleEnabled: false
-      sidecar: false
-    kiali:
-      enabled: false
-    tracing:
-      enabled: false
-    prometheus:
-      enabled: false
-    grafana:
-      enabled: false
-    sidecarInjectorWebhook:
-      enabled: false
----
-apiVersion: maistra.io/v1
 kind: ServiceMeshMemberRoll
 metadata:
   name: default
+  namespace: ${SERVICEMESH_NAMESPACE}
 spec:
   members:
-  - serving-tests
-  - serving-tests-alt
-  - knative-serving
+  - ${SERVING_NAMESPACE}
+  - ${EVENTING_NAMESPACE}
 EOF
 
   # Wait for the ingressgateway pod to appear.
-  timeout_non_zero 900 '[[ $(oc get pods -n istio-system | grep -c istio-ingressgateway) -eq 0 ]]' || return 1
+  timeout 900 '[[ $(oc get pods -n $SERVICEMESH_NAMESPACE | grep -c istio-ingressgateway) -eq 0 ]]' || return 1
```

Comment:
timeout_non_zero is the function defined in the script, 
timeout is a coreutils utility (man 1 timeout, ), that would interpret the '[[ $...' as a command which doesn't exist)

Reply:
that is a common mistake :-)